src/components/ui/ColorButtons.scss

```diff
@@ -0,0 +1,33 @@
+.colorbuttons{
```

Comment:
```suggestion
.colorbuttons {
```
I would keep the formatting and put a space here.
(and separate the words with dashes)

Reply:
refactored this!